src/Microsoft.CodeQuality.Analyzers.Exp/Setup/source.extension.vsixmanifest

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="81f388f0-1777-4c56-be8f-51f5f420037d" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>Experimental Microsoft.CodeQuality Analyzers</DisplayName>
+    <Description xml:space="preserve">Experimental Microsoft.CodeQuality Analyzers</Description>
+  </Metadata>
+  <Installation>
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0.27130.0,]" />
+  </Installation>
+  <Dependencies>
+    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
+  </Dependencies>
+  <Assets>
+    <Asset Type="Microsoft.VisualStudio.Analyzer" Path="Microsoft.CodeQuality.Analyzers.Exp.dll" IsOptionBased="true" />
```

Comment:
Unrelated to your PR, but `IsOptionBased="true"` was part of a previous prototype, that was abandoned. We should remove this property.